include/aspect/postprocess/visualization.h

```diff
@@ -269,6 +269,26 @@ namespace aspect
           std::pair<std::string, Vector<float> *>
           execute () const = 0;
       };
+
+
+      /**
+       * This class is a tag class: If a visualization postprocessor is derived
+       * from it, then this is interpreted as saying that the class will only
+       * be used to generate graphical output on the surface of the model,
+       * rather than for the entire domain.
+       */
+      template <int dim>
+      class SurfaceOnlyVisualization
```

Comment:
with "surface" you mean the boundary of the mesh, not necessarily the surface in the earth science sense, right? Worth clarifying?

Reply:
Ah, that's a good question. It really is the entire "boundary" right now.

I was about to change this to `BoundaryOnly`, but I've just changed my mind -- let me leave it as is right now, and then I'll re-work it in a follow-up to allow selecting boundary indicators on which to actually create output. There is probably no need to have much information output on the lateral boundaries.

In other words: let's leave it as is for now, and I'll get back to that later!